vendor-extra/LiberoPageBundle/src/ViewConverter/ItemRefParagraphVisitor.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Libero\LiberoPageBundle\ViewConverter;
+
+use FluentDOM;
+use FluentDOM\DOM\Element;
+use GuzzleHttp\ClientInterface;
+use Libero\ViewsBundle\Views\SimplifiedVisitor;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use Libero\ViewsBundle\Views\ViewConverterVisitor;
+use Psr\Http\Message\ResponseInterface;
+
+final class ItemRefParagraphVisitor implements ViewConverterVisitor
+{
+    use SimplifiedVisitor;
+
+    private $client;
+    private $converter;
+
+    public function __construct(ClientInterface $client, ViewConverter $converter)
+    {
+        $this->client = $client;
+        $this->converter = $converter;
+    }
+
+    protected function doVisit(Element $object, View $view) : View
```

Comment:
`View` is an immutable object used to read information like the template and the context, so much better than a mutable one. Unclear how `$view->getTemplate()` and `$this->expectedTemplate()` interact, if they do

Reply:
(answer: they must be identical)